shoop/front/middleware.py

```diff
@@ -126,7 +126,7 @@ def process_view(self, request, view_func, *view_args, **view_kwargs):
             return maintenance_response
 
     def _get_maintenance_response(self, request, view_func):
-        if "login" in view_func.__name__:
```

Comment:
I think the `login` heuristic would be good to have there, for third-party apps that likely use a view named "login", in addition to the new flag.